README.md

```diff
@@ -1,96 +1,209 @@
+        <img src="https://img.shields.io/twitter/follow/logseq?color=%2385c8c8&label=%40logseq&logo=twitter&logoColor=%2385c8c8&style=for-the-badge"
+            alt="follow on Twitter"></a>
+</p>
 
-## [Download our free Desktop app](https://github.com/logseq/logseq/releases)
-[Sponsor our contributors on Open Collective](https://opencollective.com/logseq), Logseq will move to Stripe later!
+<!-- dev badges -->
+<p align="center">
+    <a href="https://github.com/logseq/logseq/graphs/contributors" alt="Contributors">
+        <img src="https://img.shields.io/github/contributors/logseq/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="#🫶-support-logseq-development" alt="Backers on Open Collective">
+        <img src="https://img.shields.io/opencollective/backers/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="#💎-sponsors" alt="Sponsors on Open Collective">
+        <img src="https://img.shields.io/opencollective/sponsors/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="https://github.com/logseq/logseq/blob/master/LICENSE.md" alt="Activity">
+        <img src="https://img.shields.io/github/license/logseq/logseq?color=%2385c8c8&style=for-the-badge"/></a>
+    <a href="https://github.com/logseq/logseq/releases">
+        <img src="https://img.shields.io/github/v/release/logseq/logseq?color=%2385c8c8&style=for-the-badge"
+            alt="latest release version"></a>
+</p>
 
-## Why Logseq?
+## 🤔 Why Logseq?
 
-[Logseq](https://logseq.com) is a platform for knowledge management and collaboration. It focuses on privacy, longevity, and [user control](https://www.gnu.org/philosophy/free-sw.en.html).
+Logseq is a **comprehensive tool for knowledge management** and collaboration that helps you stay connected with the most critical information. With support for multiple file formats, including **Markdown**** and **Emacs Org-mode**, and a range of features for structuring and organizing your notes, Logseq is a powerful and flexible choice for anyone looking to improve their knowledge management and collaboration skills.
 
-The server will never store or analyze your private notes. Your data are plain text files and we currently support both Markdown and Emacs Org mode (more to be added soon).
+[Logseq](https://logseq.com) stores your notes locally on your device in plain text files, ensuring that your data is **private** and **secure**. Logseq also strongly emphasizes [**user control**](https://www.gnu.org/philosophy/free-sw.en.html), allowing you to access and manage your notes confidently.
 
-In the unlikely event that the website is down or cannot be maintained, your data is, and will always be yours.
+In addition to its core features, Logseq has a growing ecosystem of plugins and themes that enable a wide range of workflows and customization options. Mobile apps are also available, providing access to most features from the desktop application.
```

Comment:
most features from the desktop application -> most of the features of the desktop application

Reply:
Both are grammatically correct. I prefer to be less wordy.